src/View/Form/FormContext.php

```diff
@@ -33,17 +35,31 @@ class FormContext implements ContextInterface
      */
     protected $_form;
 
+    /**
+     * Validator name.
+     *
+     * @var string|null
+     */
+    protected $_validator = null;
+
     /**
      * Constructor.
      *
      * @param array $context Context info.
+     *
+     * Keys:
+     *
+     * - `entity` The Form class instance this context is operating on.
+     * - `validator` Optional name of the validation method to call on the Form object.
```

Comment:
We need a better format for these in general, but I don't have a suggestion right now.